### PR TITLE
HPCC-14081 New timings page using WUDetails

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes HPCC-14081

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [x] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [x] The component(s) render as expected

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq Please Review

**Comment by GordonSmith** [comment]
@jakesmith Can you test to see if it fixes your initial graph layout issue?

**Comment by jakesmith** [comment]
Have test this and seems to have resolved the 2 problems I was having
1) using CTRL confused left-click context, tooltips no longer worked
2) intermittent layout issue

**Comment by jakesmith** [comment]
back go @miguelvazq for code review

File: esp/src/eclwatch/Graph7Widget.js [hunk 1]
```diff
@@ -100,12 +100,16 @@ define([
...
                     .target(this.id + "GraphContentPane")
                     .baseUrl("")
-                    .render(function(w) {
+                    .render(function (w) {
                         w
                             .wuid(params.Wuid)
                             .graphID(params.GraphName)
                             .subgraphID(params.SubGraphId)
-                            .render()
+                            .render(function(w) {
```
Comment (miguelvazq) [inline esp/src/eclwatch/Graph7Widget.js]: Minor: You adopting syntax above in previous .render method? If so, missing space here.

File: esp/src/lws.config.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const debugServerIP = "192.168.99.103";
-const debugHPCC_JS = false; //  Should never be TRUE in a PR  ---
+const debugHPCC_JS = true; //  Should never be TRUE in a PR  ---
```
Comment (miguelvazq) [inline esp/src/lws.config.js]: This value should be false in a PR?

Comment (GordonSmith) [reply esp/src/lws.config.js]: Indeed - I will fix!

**Comment by GordonSmith** [comment]
@miguelvazq changes made as requested,

**Comment by miguelvazq** [comment]
Looks good.

**Pull request merged**